src/test/java/bio/terra/service/dataset/SelfHostedDatasetIntegrationTest.java

```diff
@@ -395,10 +307,15 @@ private void testSelfHostedDatasetLifecycle(String ingestBucket, boolean dedicat
         is(true));
   }
 
+  /**
+   * Replace every instance of ${`key`} with `value` in `inputString` for all entries in
+   * `replacements` map.
+   */
   private String replaceVars(String inputString, Map<String, String> replacements) {
```

Comment:
There is likely a way to do this offered by some other package.  But I did improve the existing implementation, which could have inadvertently allowed partial matches.